### PR TITLE
Add util-linux package

### PR DESCRIPTION
**Pull request by alanoe**
RPM specification file obtained from util-linux source rpm
in PowerKVM 3.1.0.

### CODE REVIEW

**Comment by alanoe** [comment]
Build failed because of incorrect setup in team's internal Jenkins server. @fabianorosas fixed it, I reexecuted the build and it worked: https://host-os-jenkins.aus.stglabs.ibm.com/job/build_host_os/454/console

File: util-linux/centOS/7.2/util-linux.spec [hunk 1]
```diff
@@ -0,0 +1,2889 @@
+### Header
+Summary: A collection of basic system utilities
+Name: util-linux
+Version: 2.23.2
+%define ibm_release %{?repo}.1
+Release: 33%{?dist}%{?ibm_release}
```
Comment (olavphibm) [inline util-linux/centOS/7.2/util-linux.spec]: The release should not have the "ibm_release" string. Remove it, along with the `%define` above.
See https://github.com/open-power-host-os/versions/commit/a213c4f295d611dbb62a0427718442e515654ba3, for example.

File: util-linux/util-linux.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Package:
+ name: 'util-linux'
+ clone_url: 'git://git.kernel.org/pub/scm/utils/util-linux/util-linux.git'
```
Comment (olavphibm) [inline util-linux/util-linux.yaml]: Please use the new sources format.

**Comment by mopsfelder** [comment]
We will use the utili-linux available on CentOS 7.3 as-is.

**Pull request closed without merging**